griptape/structures/structure.py

```diff
@@ -24,7 +24,7 @@ class Structure(ABC):
     id: str = field(default=Factory(lambda: uuid.uuid4().hex), kw_only=True)
     rulesets: list[Ruleset] = field(factory=list, kw_only=True)
     rules: list[BaseRule] = field(factory=list, kw_only=True)
-    tasks: list[BaseTask] = field(factory=list, kw_only=True)
+    _tasks: list[BaseTask | list[BaseTask]] = field(factory=list, kw_only=True, alias="tasks")
```

Comment:
based on the property below, this should still just be `list[BaseTask]` no?

Reply:
This is the input to the Structure, it needs to be `llist[BaseTask | list[BaseTask]]` to allow for:
```python
workflow = Workflow(
    tasks=[
        [
            PromptTask("parent", id=f"parent_{i}"),
            PromptTask("child", id=f"child_{i}", parent_ids=[f"parent_{i}"]),
        ]
        for i in range(3)
    ],
)
```